crmsh/corosync.py

```diff
@@ -811,20 +811,48 @@ class IPAlreadyConfiguredError(Exception):
     pass
 
 
-def add_node_ucast(IParray, node_id=None):
-
-    f = open(conf()).read()
-    p = Parser(f)
-
-    # to check if the same IP already configured
+def find_configured_ip(IParray):
```

Comment:
What is this IParray?
Is it a list with ip addresses? If this is a variable, we should change `IParray` if `iparray`. Otherwise, it looks like a constant, and I don't know if this is the case. It would make the code more readble

Reply:
Changed as `ip_list`